src/app/providers/config/EmbeddedConfigContextProvider.tsx

```diff
@@ -1,14 +1,28 @@
-import React from "react";
-import { ConfigContext } from '@rhoas/app-services-ui-shared';
+import React from 'react';
+import { Config, ConfigContext } from '@rhoas/app-services-ui-shared';
 import config from '../../../../config/config.json';
-import { EnvironmentConfigs, filterConfig } from "@app/providers/config/utils";
+import { EnvironmentConfigs, filterConfig } from '@app/providers/config/utils';
 
+declare const __webpack_public_path__: string;
 
 export const EmbeddedConfigProvider: React.FunctionComponent = ({ children }) => {
-  const c = config.config as EnvironmentConfigs;
-  return (
-    <ConfigContext.Provider value={filterConfig(c, config.federatedModules)}>
-      {children}
-    </ConfigContext.Provider>
-  );
-}
+  console.log('start EmbeddedConfigProvider');
+
+  const [value, setValue] = React.useState<Config | undefined>(undefined);
+
+  React.useEffect(() => {
+    const loadConfig = async () => {
+      console.log('EmbeddedConfigProvider.useEffect');
+      const c = filterConfig(config.config as EnvironmentConfigs, config.federatedModules);
+      if (c.forceConfigFetch) {
+        const r = await fetch(`${__webpack_public_path__}config.json`);
```

Comment:
Don't we already have this logic in the other config provider? Can we share the code.

Reply:
The ConfigContextProvider is not used so maybe we can even remove it completely and leave this logic only here

Reply:
ok.